apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -90,6 +89,56 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             isRequired: true,
         },
 
+        formula: {
```

Comment:
It looks like it’s already static? And it’s always balance + charge + recalculation + privilege + penalty ?